services/cloudfront/pom.xml

```diff
@@ -56,5 +56,11 @@
             <artifactId>protocol-core</artifactId>
             <version>${awsjavasdk.version}</version>
         </dependency>
+        <dependency>
+            <groupId>software.amazon.awssdk</groupId>
+            <artifactId>s3</artifactId>
+            <version>2.18.8-SNAPSHOT</version>
```

Comment:
You have a snapshot version hardcoded here instead of ${awsjavasdk.version}

Reply:
replaced